src/controllers/users.controller.ts

```diff
@@ -0,0 +1,18 @@
+import { BaseUserDto } from '@/dtos';
+import UserService from '@/services/users.service';
+import { NextFunction, Request, Response } from 'express';
+
+class UsersController {
+  public usersService = new UserService();
```

Comment:
prob should make this private too. Make sure you go over your changes thoroughly before you make a pr.